classes_practice.py

```diff
@@ -0,0 +1,43 @@
+# Создайте класс Student, который содержит атрибуты: фамилия и инициалы,
+# номер группы, успеваемость (dict из элементов).
+# Создать класс Group с атрибутом name,
+# Свяжите эти 2 класса путем инициализации класса Group в классе Student
+
+class Group():
+    def __init__(self, name):
+        self.name = name
+
+class Student():
+    def __init__(self, surname, name, group, progress):
+        self.surname = surname
+        self.name = name
+        self.group = Group(group)
+        self.progress = progress
+
+# Создать 5 студентов с разными именами, с любым количеством предметов, с оценками 1-5.
+# Они могут быть студентами одной группы.
+# Создать класс генератор результатов, который будет принимать на вход данные о студентах
+# и иметь возможность вывода фамилий и номеров групп студентов, имеющих оценки, равные только 4 или 5.
+# Метод создания экземпляров класса Students поместите в один из методов класса генератора результатов.
+
+class UserInfoAggregator():
+    def __init__(self, data_context):
+        self.students = data_context
+
+    def studentmaker(self):
```

Comment:
each word in the name should be split by `_` (e.g student_maker()). Also, the method name should answer the question What to do? (e.g create_student())